h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
> > On the other hand, I thought (and still think) that there is a clear use-case for the user wanting to find an annotation inside the document, starting from the annotation card. Do you see a problem with that use-case?

> Yes. It inverts the primary-secondary relationship of the document and its annotations. It discourages reading. It confuses two different uses for annotation and thereby muddles our product.

I totally *don't* subscribe to the idea that the annotations are necessarily secondary compared to the parent document, or that the users should be forced to read the document in order to find annotations.

I see perfectly valid use cases when one only wants to look at the annotations. (See, for example, when one is reviewing change requests on a document.)

  * * *

Anyway, I propose that we move this elsewhere, since I don't think this discussion belongs to this PR.